ruby/tests/object_cache_test.rb

```diff
@@ -48,12 +45,9 @@ def test_multithreaded_access
   end
 end
 
-if RUBY_PLATFORM == "java"
-  return
-end
-
 class ObjectCacheTest < Test::Unit::TestCase
   def create
+    omit('OBJECT_CACHE not defined') unless defined? Google::Protobuf::OBJECT_CACHE
```

Comment:
feels weird that you put `omit` on the factory method.  You sure that is correct?

Reply:
Under the hood, `omit` raises an exception that test unit catches and puts in a different category than other unchecked exceptions. I put it here just to avoid code echo, but I don't feel strongly about keeping the test DRY.